src/askContainer/commands/viewAllSkills.ts

```diff
@@ -24,6 +25,10 @@ const ALL_SKILLS_QUICK_PICK_TITLE = 'All skills';
 const SKILL_RETRIEVAL_PLACEHOLDER = 'Getting the list of skills...';
 const SKILL_NOT_FOUND = 'No Alexa custom skills found under profile. Please create one first.';
 const SKILL_RETRIEVAL_FAILED = 'Alexa skills retrieval failed.';
+const VIEW_ALL_SKILLS_LAST_UPDATE_TIME = 'viewAllSkillsLastUpdateTime';
+const NUMBER_OF_INTERVALS = 1;
+const HOUR = 3600000;
+const MINUTE = 60000;
```

Comment:
I am guessing you wanted to add this constant for the scheduler timer on the list skills fetch command, as mentioned in your PR description. Did you miss adding that logic to the PR?

Reply:
I left this for testing convenience, so we can test by `MINUTE` instead of `HOUR`. I will remove it in the next revision.